src/app/hooks/useViewTracker/index.test.jsx

```diff
@@ -0,0 +1,182 @@
+/* eslint-disable no-template-curly-in-string */
+import React from 'react';
+import { renderHook, act } from '@testing-library/react-hooks';
+import { useInView } from 'react-intersection-observer';
+
+import { ServiceContextProvider } from '#contexts/ServiceContext';
+import { RequestContextProvider } from '#contexts/RequestContext';
+import * as atiUrl from '#containers/ATIAnalytics/atiUrl';
+import useViewTracker from '.';
+
+import { CORRESPONDENT_STORY_PAGE } from '#app/routes/utils/pageTypes';
```

Comment:
Any reason you chose this page type for the test, it's a not that common page type compared to a standard `STORY_PAGE` page, I wonder if it might be worth testing with the majority case?

Reply:
We were confused by that page type. Is it a CPS page type but with incorrect full naming? Anyway, there currently isn't a `STORY_PAGE` ATI data transformation function thingy setup here https://github.com/bbc/simorgh/blob/latest/src/app/containers/ATIAnalytics/params/index.js#L26-L53 but we will have to create this when we come to using this custom hook - which IMO is outside the scope of this issue.

Reply:
It is basically a Story page with a byline at the top, so a Story by a specific journalist basically. Yeah, the standard `STORY_PAGE` method has never been wired up, I did it in two lines in my event despatch PR: https://github.com/bbc/simorgh/pull/9063/files#diff-52471018ee0813e7f9bac2e8bf195651962ee4071a1475be8824cd86418d3183R27

Tbh, I think the metadata is the same so feel free to leave it

Reply:
ah cool. I thought it would have to be a brand new function. I'll add those changes to this PR. Thanks